src/plonetheme/tokyo/browser/templates/portal_header.pt

```diff
@@ -5,9 +5,8 @@
     <div class="container-fluid header-wrapper">
 
       <a class="header-logo" href="${navroot}">
-        <img src="" alt="" tal:attributes="src string:${navroot}/++plone++plonetheme.tokyo/plone_logow.svg">
+          <img src="${python: view.logo if view.logo else navroot+'/++plone++plonetheme.tokyo/plone_logow.svg'}" alt="">
```

Comment:
Indent is now ok or to big? We should use Site Title for alt and title tag as it's rendered on default Plone.

Reply:
Done!